content/en/docs/installation/supported-releases.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Supported releases"
+linkTitle: "Supported releases"
+weight: 5
+type: "docs"
+---
+
+<!--
+Inspired by https://istio.io/latest/about/supported-releases/
+-->
+
+This page lists the status, timeline and policy for currently supported
+releases. Supported releases of cert-manager include releases that are in
+the active maintenance window and are patched for security and bug fixes.
+Subsequent patch releases on a minor release do not contain backward
+incompatible changes.
+
+## Support status of cert-manager releases
+
+In the below table, the cert-manager versions are shown in the leftmost
+column. The versions of Kubernetes are shown in the rightmost columns, and
+"yes" means that this version of cert-manager can work with this version of
+Kubernetes. The "~" sign is used for expected date, since the actual date
+might be pushed a few weeks later. The "\*" sign before a Kubernetes version
+means that this Kubernetes version is one of the three Kubernetes versions
+[currently
+supported](https://kubernetes.io/docs/setup/release/version-skew-policy/).
+The "EOL" abbreviation above stands for End Of Life.
+
+| Version   | Support |     Release Date     |         EOL          | Supported Kubernetes versions |
+| --------- | ------- | :------------------: | :------------------: | :---------------------------: |
+| master    | No      |                      |                      |                               |
+| [v1.3][]  | N/A     | _[~Apr 8, 2021][tl]_ |  _[~Aug 2021][tl]_   |                               |
+| [v1.2][]  | **Yes** |     Feb 10, 2021     |  _[~Jun 2021][tl]_   |          1.16 → 1.20          |
+| [v1.1][]  | **Yes** |     Nov 24, 2021     | _[~Apr 8, 2021][tl]_ |          1.11 → 1.20          |
+| [v1.0][]  | No      |     Sep 2, 2020      |     Feb 10, 2021     |          1.11 → 1.20          |
+| [v0.16][] | No      |     Jul 23, 2020     |     Nov 24, 2021     |          1.11 → 1.20          |
+| [v0.15][] | No      |     May 6, 2020      |     Sep 2, 2020      |          1.11 → 1.20          |
+| [v0.14][] | No      |     Mar 11, 2020     |     Jul, 23 2020     |          1.11 → 1.20          |
+| [v0.13][] | No      |     Jan 21, 2020     |     May, 6 2020      |          1.11 → 1.20          |
+| [v0.12][] | No      |     Nov 27, 2019     |     Mar 11, 2020     |          1.11 → 1.20          |
+| [v0.11][] | No      |     Oct 10, 2019     |     Jan 21, 2020     |          1.9 → 1.20           |
```

Comment:
I'm not sure about the supported Kubernetes versions

Reply:
Is it the case that we need to sync our supported Kubernetes versions with upstream?
For example, right now when I deploy `cert-manager` on k8s v1.19 I get this in logs 
```
W0329 20:23:26.664405       1 warnings.go:67] networking.k8s.io/v1beta1 Ingress is deprecated in v1.19+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress
```

Maartje says [here](https://github.com/jetstack/cert-manager/pull/3499) that we can start using `networking.k8s.io/v1` once `cert-manager`'s minimum supported k8s version is v1.19 and I guess we have to do it before we start supporting v1.22, so I guess we wouldn't, for example, be able to support v1.18 and v1.22 simultaneously?
Perhaps always the latest 3 or 4 minor releases?

Reply:
For older versions which aren't supported today, is it worth leaving out the supported k8s versions info and just putting "N/A"? There's no realistic guarantee that, say, v0.11 is gonna work on k8s 1.20, and nobody is going to be running tests to check that.

By the time this PR is merged, the supported versions will be v1.3 and v1.2 - I think we should document which versions of k8s we support for those two versions, and leave the rest blank.

Reply:
My understanding is 

K8S 1.19, the support window is 1 year, so 1.19, 1.20, 1.21 will received patch updates for 1 year after their release.

K8S <= 1.18, the support window is 9 months, so 1.18, 1.17 and 1.16 are no longer supported.
(although they do still seem to be patching 1.18)

So I think we should only run our tests on 1.19, 1.20 and 1.21.

https://kubernetes.io/blog/2020/08/31/kubernetes-1-19-feature-one-year-support/

Although when I look at the tag dates for those older versions, it doesn't really match the 9 month policy:

1.16:
 * https://github.com/kubernetes/kubernetes/tree/v1.16.0 (13 Sep 2019)
 * https://github.com/kubernetes/kubernetes/tree/v1.16.15 (2 Sep 2020)

1.17:
 * https://github.com/kubernetes/kubernetes/tree/v1.17.0 (6 Dec 2019)
 * https://github.com/kubernetes/kubernetes/tree/v1.17.17 (13 Jan 2021)

1.18:
 * https://github.com/kubernetes/kubernetes/tree/v1.18.0 (25 Mar 2020)
 * https://github.com/kubernetes/kubernetes/tree/v1.18.17 (18 Mar 2021)

Reply:
AKS release and supported version schedule: https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions#aks-kubernetes-release-calendar

GKE supported versions: https://cloud.google.com/kubernetes-engine/docs/release-notes

Reply:
Kubernetes Yearly Support Period  KEP: https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/1498-kubernetes-yearly-support-period

Reply:
@wallrj Could we have a quick call about that? I feel a bit lost with all these versions and I don't know what I should be writing anymore 😅

Reply:
Note: our current coverage still does not test Kubernetes 1.21.

| Release branch | Prow configuration            | testgrid                | Kubernetes versions tested   | Periodicity |
| :------------: | :---------------------------- | :---------------------- | :--------------------------- | :---------: |
|     master     | [`next-periodics.yaml`][]     | [`testgrid-next`][]     | 1.16, 1.17, 1.18, 1.19, 1.20 | On each PR  |
|  release-1.3   | [`previous-periodics.yaml`][] | [`testgrid-previous`][] | 1.16, 1.17, 1.18, 1.19, 1.20 | Every night |
|  release-1.2   | N/A                           |                         | N/A                          |     N/A     |

Reply:
I'll resolve this comment for now, although I wish I didn't because this thread is very informative